apps/hip/static/js/common.js

```diff
@@ -0,0 +1,9 @@
+export function mobileModalOpen (target) {
```

Comment:
Minor comments, but you don't have to accept them. I think I'd prefer these to be named `openMobileModal` and `closeMobileModal` which makes it clear to me that by calling this function I am going to perform that action. A comment here saying something like `// target: element to open as a modal` or something might be helpful as well.

Reply:
done